Atlassian.JiraSoftwareLTS version 10.3.10

noyard opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/290836)

noyard committed:
Atlassian.JiraSoftwareLTS version 10.3.10

stephengillie commented:
Verify:
<img width="993" height="710" alt="image" src="https://github.com/user-attachments/assets/635c7607-bb0d-411c-9868-426180270740" />
<img width="451" height="208" alt="image" src="https://github.com/user-attachments/assets/854e7e67-b661-4857-bebb-60656e04ced5" />

Hi @noyard,

I'm not sure that this application passes the scripted application check. While there is a service that gets installed, it doesn't appear to be the main executable for this application. To launch the application and reach the setup page above, one batch file needs to be launched, which launches another, etc. What are your thoughts on this?

noyard committed:
removed dependancy.

Removed JDK21 - per Atlassian If you use the Windows or Linux installer, Jira includes and configures its own bundled JRE, so you don’t need to install Java separately.
